src/Components/TitleAutocomplete.js

```diff
@@ -35,15 +45,32 @@ export default function TitleAutocomplete({ search, setShowSearch }) {
     if (key === "Escape") setShowSearch(false);
   }
 
+  // Prevents popper from showing "No Results" during loading state and while there's no search term
+  useEffect(() => {
```

Comment:
do we want a set of options when the user clicks in the box with no query?  Also, do we want the popper to flash off and on again all the time as they add to their query and more requests get sent?  I might suggest keeping the popper open the whole time.  Instead, maybe use `keepPreviousData` in the useQuery options for search queries (to keep old results up while a new query is in flight), and you can then use `searchLoading` to detect the very first load before any data has been returned.  I haven't actually tried this out, so maybe there are bugs I am not predicting, but I feel like this would be a pretty smooth experience that would match other search boxes (google, yt, etc)

Reply:
I implemented essentially what you describe.  Note, I added in a 'loading' state for the popper when useQuery is making its call. 

Also, to eliminate a couple of unwanted "flashes" I (1) use the data from useQuery **_directly_** in our Autocomplete component, and (2) Combined the useQuery 'isFetching' state into one loading variable to rule them all!